GTViewController/Classes/Nav Classes/GTFollowupViewController.m

```diff
@@ -321,6 +329,16 @@ - (BOOL)validateFields {
     return YES;
 }
 
+- (NSString *)currentLanguageCode {
```

Comment:
The property statement already generates getters and setters for you. If you want to access it as currentLanguageCode I would update your property statement